js/src/lib/Details/Popularity.js

```diff
@@ -76,6 +99,7 @@ class Popularity extends Component {
               humanDependents={humanDependents}
               name={name}
             />
+            <JsDelivrHits jsDelivrHits={jsDelivrHits} />
```

Comment:
I think It would make sense to have that number (a) in the sidebar where the jsdelivr url is, or (b) in a new thing for usage, where jsdelivr is one of the keys and unpkg etc. also are available. Wdyt?

Reply:
I don't think this metric is more important than the npm downloads, and now that I looked at it, last 30 days is probably too long to fit on that line. To me it looks good, but would just like the input of @mjackson of unpkg to see if he'd rather see unpkg on there too :)

Reply:
>I don't think this metric is more important than the npm downloads

I meant below downloads, above dependents. You might be right about the text length issue though 😄

Reply:
![image](https://user-images.githubusercontent.com/6192491/46402415-ef62f480-c6ff-11e8-802a-d0c45c42c440.png)

Reply:
maybe it's easier to read if it's just `jsDelivr last 30 days`